google/resource_compute_backend_service_migrate_test.go

```diff
@@ -0,0 +1,95 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestComputeBackendServiceMigrateState(t *testing.T) {
+	cases := map[string]struct {
+		StateVersion       int
+		Attributes         map[string]string
+		ExpectedAttributes map[string]string
+		Meta               interface{}
+	}{
+		"v0 to v1": {
+			StateVersion: 0,
+			Attributes: map[string]string{
+				"backend.#":                         "1",
+				"backend.242332812.group":           "https://www.googleapis.com/compute/v1/projects/project_name/zones/zone_name/instances/instanceGroups/igName",
```

Comment:
You could add also a test with a beta group. Even though it wouldn't add much value I guess...

Reply:
Eh, not going to bother on this one since this test doesn't make any API calls or really look at the version